packages/flutter/lib/src/rendering/debug.dart

```diff
@@ -2,11 +2,15 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:ui' as ui;
 import 'dart:async';
 import 'dart:collection';
+import 'dart:convert' show JSON;
+import 'dart:developer';
```

Comment:
maybe import this "as" something so that the "log" call ends up being "developer.log" or some such?